lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -31,7 +31,16 @@ module.exports = {
 
   createStack() {
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
-
+    if (/^[^a-zA-Z].+|.*[^a-zA-Z0-9\-].*/.test(stackName) || stackName.length > 128) {
+      const errorMessage = [
+        `The stack name "${stackName}" is not quallify. `,
+        'A stack name can contain only alphanumeric',
```

Comment:
Same here. Would be great if it's something like `A service name should only contain alphanumeric (case sensitive) characters and hyphens`.